pinot-core/src/main/java/org/apache/pinot/core/operator/ExecutionStatistics.java

```diff
@@ -22,8 +22,13 @@
  * The <code>ExecutionStatistics</code> class contains the operator statistics during execution time.
  */
 public class ExecutionStatistics {
+  // The number of documents scanned post filtering.
   private long _numDocsScanned;
+  // The number of entries (single value entry contains 1 entry, multi-value entry may contain multiple entries) in the
+  // filtering phase of the query execution: could be larger than the total scanned doc num because of multiple
+  // filtering predicates and multi-value entry.
```

Comment:
The number of entries (single-value entry contains 1 **value**, multi-value entry may contain multiple **values**) **scanned** in the filtering phase of the query execution. Could be larger than the number of documents scanned because of multiple filtering predicates.

Reply:
Last sentence also changed (multi-value entry is not the reason)

Reply:
I thought one reason why number_of_entries_scanned > number_of_doc_scanned is that for a multi-value entry, each value in it could count as one entry scanned. Is it right?

Reply:
Oh sorry my bad, you are right. The semantic for this value changed with #4455 and I totally forgot about it...
So right now the entry has different meaning in `_numEntriesScannedInFilter` (each MV value contains multiple entries) and `_numEntriesScannedInFilter` (both SV and MV value are counted as one entry).
Can you please add a TODO to make entry semantic the same in `_numEntriesScannedInFilter` and `_numEntriesScannedInFilter`? Currently it is very misleading.
I think it might be better to add a new stats `_numValuesScannedInFilter` to replace the current `_numEntriesScannedInFilter`, but it is out of the scope of this pr.

Reply:
TODO added. Thanks for the clarification.